.storybook/main.ts

```diff
@@ -3,7 +3,11 @@ import type { StorybookConfig } from '@storybook/nextjs';
 
 const config: StorybookConfig = {
   stories: ['../components/**/*.stories.tsx'],
-  addons: ['@storybook/addon-controls', '@storybook/addon-interactions'],
+  addons: [
+    '@storybook/addon-controls',
+    '@storybook/addon-interactions',
+    'storybook-addon-manual-mocks',
```

Comment:
Why we need the manual mocks addon?

Reply:
Manual mock addon helps to pick the mock instead of real service/hook during story execution. By doing this the story will be uniform across all the platforms. The implementation worked earlier but now it is failing as absolute paths will not work with this addon and we have made absolute imports mandatory while linting. Working on alternative solution